### PR TITLE
Finish general changes

### PR DESCRIPTION
**Pull request by yirenlu92**

### CODE REVIEW

File: runtime/program_lifecycle.md [hunk 1]
```diff
@@ -98,3 +111,95 @@ In other words, you can use `addEventListener` to register multiple `"load"` or
...
+About to exit...
+Exiting
+```
+
+This has allowed us to polyfill `process.on("beforeExit")` in the Node
+compatibility layer.
+
+## `unhandledrejection` event Example:
+
+This release adds support for the unhandledrejection event. This event is fired
```
Comment (kt3k) [inline runtime/program_lifecycle.md:171]: The first sentence seems unnecessary here to me.

File: runtime/program_lifecycle.md [hunk 2]
```diff
@@ -98,3 +111,95 @@ In other words, you can use `addEventListener` to register multiple `"load"` or
...
+    at new Foo (file:///dev/unhandled_rejection.js:7:29)
+    at file:///dev/unhandled_rejection.js:10:1
+} reason: Error: bar not available
+    at new Foo (file:///dev/unhandled_rejection.js:7:29)
+    at file:///dev/unhandled_rejection.js:10:1
+unhandled rejection at: Promise { <rejected> undefined } reason: undefined
+```
+
+This API will allow us to polyfill `process.on("unhandledRejection")` in the
+Node compatibility layer in future releases.
```
Comment (kt3k) [inline runtime/program_lifecycle.md:205]: The support of `process.on("unhandledRejection")` in Node compat layer is already available (https://github.com/denoland/deno_std/pull/2460). I think this sentence should be dropped.

File: tools/init.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Starting a new project
+
+Starting a new project with Deno has always been incredibly simple: you just
+need a single file to get going. No need for any configuration files, dependency
+manifests, or build scripts.
+
+Users coming from other ecosystems are often not used to this simplicity - they
+often look for a tool to scaffold out a basic project structure to get them
+started on the right path. In this release we have added a deno init subcommand
+that scaffolds a basic Deno project.
```
Comment (kt3k) [inline tools/init.md:10]: The phrase `In this release` doesn't make sense here. How about rephrasing this like:
```suggestion
started on the right path. `deno init` subcommand scaffolds a basic Deno
project.
```

**Comment by kt3k** [comment]
Other apart looks good to me

**Review by ry — APPROVED** [review]
Thanks Ren!

**Pull request merged**